frontend/src/scenes/funnels/FunnelBarGraph.tsx

```diff
@@ -308,12 +308,7 @@ export function FunnelBarGraph({ color = 'white' }: { color?: string }): JSX.Ele
     return (
         <div
             data-attr="funnel-bar-graph"
-            className={clsx(
-                'funnel-bar-graph',
-                FunnelLayout.horizontal,
```

Comment:
We should probably get rid of this `FunnelLayout` type too (and related stuff)

Reply:
These are actually for the Left-to-right vs top-to-bottom distinction. Not for the UI.